cogs/dev.py

```diff
@@ -83,6 +93,36 @@ async def sync(self, ctx: commands.Context, guilds: commands.Greedy[discord.Obje
 
         await ctx.send(f"Synced the tree to {ret}/{len(guilds)}.")
 
+    @commands.group(aliases=['ext'])
+    async def extensions(self, ctx: commands.Context):
+        """Utilities for extensions"""
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help("ext")
+
+    @extensions.command(name="list")
+    async def ext_list(self, ctx):
+        """Lists currently loaded extensions"""
+        ext_list = "\n".join(f"- {ext}" for ext in self.bot.extensions)
+        await ctx.send(f"List of loaded extensions:\n{ext_list}")
+
+    @extensions.command(name="load")
+    async def ext_load(self, ctx, ext_name: ext_converter):
+        """Loads an extension"""
+        await self.bot.load_extension(ext_name)
+        await ctx.send(f"Successfully loaded extension `{ext_name}`")
+
+    @extensions.command(name="unload")
+    async def ext_unload(self, ctx, ext_name: ext_converter):
+        """Unloads an extensions"""
+        await self.bot.unload_extension(ext_name)
+        await ctx.send(f"Successfully unloaded extension `{ext_name}`")
+
+    @extensions.command(name="reload")
+    async def ext_reload(self, ctx, ext_name: ext_converter):
+        """Reloads an extension"""
+        await self.bot.reload_extension(ext_name)
+        await ctx.send(f"Successfully reloaded extension `{ext_name}`")
```

Comment:
Just a couple general comments here:
- Include the type for `ctx` within function parameters (`commands.Context`)
- Format the doc-strings to be like the other commands for consistency (newlines, punctuation) + typo in `"""Unloads an extensions"""`
- Add `logger.debug` statements when loading/unloading/reloading extensions (see `cogs/filter.py` for example)